module/TaskManagement/public/js/task-management.js

```diff
@@ -1,895 +1,939 @@
-			}
-		});
-	},
-	
-	skipAssignShares : function (e){
-		var modal = $(e.delegateTarget);
-		var form = modal.find("form").first();
-
-		var that = this;
-		
-		$.ajax({
-			url: form.attr('action'),
-			headers: {
-				'GOOGLE-JWT': sessionStorage.token
-			},
-			method: 'POST',
-			data: {},
-			success: function() {
-				modal.modal('hide');
-				that.listTasks();
-			},
-			statusCode: {
-				400 : function(jqHXR, textStatus, errorThrown){
-					json = $.parseJSON(jqHXR.responseText);
-					if(json.description != undefined) {
-						that.show(modal, 'danger', json.description);
-					}
-					if(json.errors != undefined) {
-						that.show(modal, 'danger', json.errors[0].message);
-					}
-				}
-			},
-			error: function(jqHXR, textStatus, errorThrown) {
-				that.show(modal, 'danger', 'An unknown error "' + errorThrown + '" occurred while trying to assign shares');
-			}
-		});
-	},
-	
-	assignShares : function (e){
-		var modal = $(e.delegateTarget);
-		var form = $(e.target)
-
-		var that = this;
-		
-		$.ajax({
-			url: form.attr('action'),
-			headers: {
-				'GOOGLE-JWT': sessionStorage.token
-			},
-			method: 'POST',
-			data: form.serialize(),
-			success: function() {
-				modal.modal('hide');
-				that.listTasks();
-			},
-			statusCode: {
-				400 : function(jqHXR, textStatus, errorThrown){
-					json = $.parseJSON(jqHXR.responseText);
-					if(json.description != undefined) {
-						that.show(modal, 'danger', json.description);
-					}
-					if(json.errors != undefined) {
-						that.show(modal, 'danger', json.errors[0].message);
-					}
-				}
-			},
-			error: function(jqHXR, textStatus, errorThrown) {
-				that.show(modal, 'danger', 'An unknown error "' + errorThrown + '" occurred while trying to assign shares');
-			}
-		});
-	},
-	
-	show: function(container, level, message) {
-		alertDiv = container.find('div.alert');
-		alertDiv.removeClass();
-		alertDiv.addClass('alert alert-' + level);
-		alertDiv.text(message);
-		alertDiv.show();
-	},
-	
-	getLabelForAssignShares: function(daysLeft){
-		
-		if(daysLeft !== null){
-			if(daysLeft == 1){
-				return ": "+daysLeft + " day left";
-			}else if(daysLeft == 0){
-				return ": less than a day";
-			}
-			return ": "+daysLeft + " days left";
-		}
-		return "";
-	}
-	
-};
-
-var TASK_STATUS = (function() {
-	var labels = {
-		0: 'Idea',
-		10: 'Open',
-		20: 'Ongoing',
-		30: 'Completed',
-		40: 'Shares assignment in progress',
-		50: 'Closed'
-	};
-	
-	var values = {
-		'IDEA':			0,
-		'OPEN':			10,
-		'ONGOING':		20,
-		'COMPLETED':	30,
-		'ACCEPTED':		40,
-		'CLOSED':		50
-	}
-	
-	return {
-		get: function(name) { return values[name]; },
-		label: function(name) { return labels[name]; }
-	};
-})();
-
-$().ready(function(e){
-	$('#content div.alert').hide();
-	$('#firstLevelMenu li').eq(0).addClass('active');
-	collaboration = new TaskManagement();
-	collaboration.listTasks();
-	collaboration.updateStreams();
+var TaskManagement = function()
```

Comment:
Questo file risulta completamente modificato (come puoi verificare in "Files changed" ogni riga è stata sostituita), ma non riesco a capire perché. Quali cambiamenti sono stati apportati?